src/components/backend-ai-console.ts

```diff
@@ -149,7 +149,7 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
     //this.mainToolbar.scrollTarget = this.appPage;
     this.drawerToggleButton = this.shadowRoot.querySelector('#drawer-toggle-button');
     this.sidebarMenu = this.shadowRoot.getElementById('sidebar-menu');
-    this.splash = this.shadowRoot.querySelector('#about-panel');
+    this.splash = this.shadowRoot.querySelector('#about-backend.ai-panel');
```

Comment:
@swapniljha001 We need to avoid using dot (.) at the selector name (even it works). please change it to backendai. :)